src/Akeneo/UserManagement/Bundle/Controller/RoleController.php

```diff
@@ -61,6 +68,13 @@ public function delete(Request $request, $id): Response
             throw $this->createNotFoundException(sprintf('Role with id %d could not be found.', $id));
         }
 
+        $minimumEditRoleRoles = $this->editRolePermissionsRoleRepository->getRolesWithMinimumEditRolePermissions();
```

Comment:
Only a suggestion but it's probably less confusing to name it like this

```suggestion
        $rolesWithMinimumEditRole = $this->editRolePermissionsRoleRepository->getRolesWithMinimumEditRolePermissions();
```